app/src/main/java/org/simple/clinic/summary/bloodsugar/view/BloodSugarSummaryView.kt

```diff
@@ -43,6 +43,8 @@ import org.simple.clinic.util.UserClock
 import org.simple.clinic.util.UtcClock
```

Comment:
All commits tend to update something or the other. This commit message can be more descriptive.